### PR TITLE
Initial attempt to convert to go sdk.

### PR DESCRIPTION
**Pull request by bmanth60**
This is to solve: https://github.com/Vertamedia/clickhouse-grafana/issues/242

### CODE REVIEW

**Comment by Slach** [comment]
;) wow @bmanth60 thanks for contributing ;)

File: datasource.go [hunk 1]
```diff
@@ -35,61 +30,19 @@ type ClickHouseMeta struct {
...
-		return nil, err
-	}
-
-	if response.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("invalid status code. status: %v", response.Status)
-	}
-
-	return parseResponse(body, refId, req.GetTimeRange())
-}
+var httpClient = &http.Client{}
```
Comment (bmanth60) [inline datasource.go:38]: this should be removed since client is now part of context

**Comment by Slach** [comment]
@bmanth60 any news about switching to new golang sdk?
currently i found
https://github.com/valeriakononenko/grafana-clickhouse-plugin/tree/master/
which already switched to new SDK
would it help you to complete your pull request?

**Comment by bmanth60** [comment]
@Slach thanks! Yeah, sorry I have been procrastinating. I have been meaning to complete this though.

**Comment by bmanth60** [comment]
Wow, that code is a beauty... valeriakononenko did a fantastic job!

**Comment by bmanth60** [comment]
@Slach when I try to startup the latest plugin from master, I get a signature validation error `Failed to load plugin" logger=plugins error="plugin \"vertamedia-clickhouse-datasource\"'s signature has been modified`, which I think is related to https://community.grafana.com/t/signed-backend-plugin-keeps-throwing-modified-signature-error/39687/6 

Have you had this problem before? I am wondering if I just need to resign it, but I don't want to make a release, just want to work on it locally. I tried to use the env to disable signatures or set the app mode to development and neither seem to allow me to test the new plugin.

Please let me know if you have had this problem before, ty!

**Comment by Slach** [comment]
@bmanth60  new sign process and grafana 7.3 and require setup `GRAFANA_API_KEY` environment

copy `.release_env.example` to `.release_env` and edit it
and say me your grafana.net account I try to add you to Vertamedia organization

**Comment by bmanth60** [comment]
@Slach cool, thanks I have created my account name as bmanth60, please add me to the org, thanks!

**Comment by Slach** [comment]
@bmanth60 which email did you use when create account on grafana.com ?

**Comment by bmanth60** [comment]
@Slach I signed up with bman.brianthai@gmail.com

**Pull request closed without merging**